validator/config.json.example

```diff
@@ -0,0 +1,20 @@
+{
+  "fiware": {
```

Comment:
I would name this section as  `dataModel` instead of `fiware`

Reply:
i used dmv for brevity